### DISCUSSION TITLE
How to specify search node for forwarder

### DISCUSSION THREAD

**Discussion by Sahale** [Security-Onion-Solutions/securityonion#4608] [Unsupported Versions] [answered]
Hi!
Is there any way to specify which search node should be used to save data from specified forwarder in distributed architecture? I.e. sensor #1 should save data to search #2 and sensor #2 to search #1? Main purpose is to save keep from different sources separately. Thanks!

**Comment by weslambert** [comment, marked as answer]
We don't support this OOB at the moment.  One option might be to use a heavy node to store the data locally to the sensor, although with higher throughput environments, we don't recommend the usage of a heavy node, and depending on the geographical location and resource utilization of the heavy node, it may cause your overall search to be slower.
